packages/app-content-pages/src/screens/About/components/HowItWorks.js

```diff
@@ -0,0 +1,310 @@
+import { Anchor, Box, Button, Heading, Image, Text } from 'grommet'
+import { useTranslation } from 'next-i18next'
+import styled from 'styled-components'
+import {
+  Certificate,
+  Compass,
+  Compliance,
+  Deploy,
+  Download,
+  Info,
+  Troubleshoot,
+  Validate
+} from 'grommet-icons'
+
+import MaxWidthContent from '@shared/components/MaxWidthContent/MaxWidthContent.js'
+import { HeadingForAboutNav } from '../About.js'
+
+const GradientBox = styled(Box)`
+  position: relative;
+  background: linear-gradient(
+    212deg,
+    rgba(0, 151, 157, 0.2) 8.04%,
+    rgba(255, 255, 255, 0.2) 11.21%,
+    rgba(0, 93, 105, 0.2) 24.51%,
+    rgba(255, 255, 255, 0.2) 45.18%,
+    rgba(0, 93, 105, 0.2) 67.56%,
+    rgba(255, 255, 255, 0.2) 85.31%,
+    rgba(0, 151, 157, 0.2) 86.6%,
+    rgba(255, 255, 255, 0) 100.98%
+  );
+
+  &:before {
+    content: '';
+    position: absolute;
+    top: 0;
+    left: 0;
+    height: 2px;
+    width: 100%;
+    background: linear-gradient(
+      90deg,
+      transparent 0%,
+      #a6a7a9 50%,
+      transparent 100%
+    );
+  }
+`
+
+const ListsContainer = styled(Box)`
+  flex-direction: row;
+
+  @media (width < 37rem) {
+    flex-direction: column;
+    align-items: center;
+    row-gap: 50px;
+  }
+`
+
+const Institute = styled(Box)`
+  border: solid white 1px;
+  width: 100%;
+  border-radius: 8px;
+  backdrop-filter: blur(10px);
+  background: rgba(239, 242, 245, 0.3);
+  box-shadow: 2px 2px 4px 0px rgba(0, 0, 0, 0.25);
+  padding: 5px;
+  align-items: center;
+`
+
+const ArrowBox = styled(Box)`
+  position: relative;
+  width: 240px; // same as ArrowSVG
+  height: 46px; // upper portion of SVG
+  align-items: center;
+  justify-content: center;
+  padding: 0 25px;
+  margin-bottom: 50px;
+`
+
+const StyledButton = styled(Button)`
+  background: white;
+  box-shadow: 2px 2px 4px 0px rgba(0, 0, 0, 0.25);
+  font-size: 0.8rem;
+  padding: 5px;
+  width: 240px; // same as ArrowSVG
+  min-width: 130px;
+  border-radius: 8px;
+  border: solid 1px ${props => props.theme.global.colors['neutral-2']};
+  text-align: center;
+
+  &:hover {
+    text-decoration: none;
+  }
+`
+
+const BoxWithConnector = styled(Box)`
+  position: relative;
+
+  &:not(:nth-child(2)) {
+    &:before {
+      content: '';
+      position: absolute;
+      bottom: calc(50% + 40px);
+      left: 24px;
+      height: 30px;
+      width: 2px;
+      background: white;
+    }
+  }
+
+  &:not(:last-child) {
+    margin-bottom: 70px;
+
+    &:after {
+      content: '';
+      position: absolute;
+      top: calc(50% + 40px);
+      left: 24px;
+      height: 30px;
+      width: 2px;
+      background: white;
+    }
+  }
+`
+
+const ArrowSVG = () => (
```

Comment:
Not sure if this is a broader conversation but my personal preference is to have all SVG's live in a folder that gets pulled in instead of scattering these assets throughout the codebase. That way there's at least some sense of reusability. May not be the case here - it's a pretty specific SVG - but a thought to consider.

Reply:
This is a good point - For now though, I'm going to leave this SVG in the HowItWorks component because it's specific to that section. If code gets reused at least twice throughout the About Pages and Get Involved pages I'll place it in a shared file, similar to the SharedStyledComponents in this package.